src/Services/OData/ODataTools.cs

```diff
@@ -82,84 +82,20 @@ internal static IEnumerable<ActionBase> GetActions(Content content, ODataRequest
                 request != null ? request.Scenario : null,
                 string.IsNullOrEmpty(backUrl) ? null : backUrl);
         }
-        internal static IEnumerable<ODataActionItem> GetActionItems(Content content, ODataRequest request)
+        internal static IEnumerable<ODataActionItem> GetHtmlActionItems(Content content, ODataRequest request)
         {
-            return GetActionsGroupedByScenario(content, request).Select(a => new ODataActionItem
+            return GetActions(content, request).Where(a => a.IsHtmlOperation).Select(a => new ODataActionItem

```

Comment:
Same here: this was the point of the whole change: serving all kinds of actions, not just html ones.